pyproject.toml

```
@@ -52,17 +52,17 @@ dependencies = [
 
 [project.optional-dependencies]
 cli = [
-    "httpx-auth ~=0.22.0",
+    "httpx-auth ~=0.23.1",
     "pyyaml ~=6.0",
     "typer ~=0.15.1",
 ]
 testing = [
     "cryptography >=43.0.0,<45",
     # Pytest runs, end in Segmentation Fault for dlite-python versions >0.5.23
-    "dlite-python <=0.5.23; python_version<'3.13'",  # DLite is not yet compatible with Python 3.13
+    "dlite-python <=0.5.27; python_version < '3.13'",  # DLite is not yet compatible with Python 3.13
```

Still getting Segmentation faults
```suggestion
    "dlite-python <=0.5.23; python_version < '3.13'",  # DLite is not yet compatible with Python 3.13
```